package-lock.json

```diff
@@ -0,0 +1,4835 @@
+{
```

Comment:
How is this file created and how is it supposed to be updated?

Reply:
It's created by `npm` and it's used by bazel to ensure reproducibility. See here: https://bazelbuild.github.io/rules_nodejs/#hermeticity-and-reproducibility